doc/source/train/user-guides/experiment-tracking.rst

```diff
@@ -1,195 +1,326 @@
-.. _train-experiment-tracking:
+.. _train-experiment-tracking-native:
 
+===================
 Experiment Tracking
 ===================
 
-Most experiment tracking libraries work out-of-the box with Ray Train. This means
-you can use the regular tracking library APIs to report your results, models, and
-other artifacts.
+.. note::
+    This guide is relevant for all DataParallelTrainer, including TorchTrainer and TensorflowTrainer.
 
-Ray Train also exposes logging callbacks that automate some of these tasks.
+Most experiment tracking libraries work out-of-the-box with Ray Train. 
+This guide provides instructions on how to set up the code so that your favorite experiment tracking libraries 
+can work for distributed data parallel training with Ray Train. We will conclude the session with debugging
+tips.
```

Comment:
Let's just call it "distributed training" - in model parallel training or a mix we also only want to report once